tests/fuzzing/oss_fuzz_build.sh

```diff
@@ -4,10 +4,10 @@
 # if run outside of it.
 # The api, compile_go_fuzzer() is provided by the OSS-fuzz
 # environment and is a high level helper function for a series
-# of compilation and linking steps to build the fuzzers in the 
+# of compilation and linking steps to build the fuzzers in the
 # OSS-fuzz environment.
 # More info about compile_go_fuzzer() can be found here:
 #     https://google.github.io/oss-fuzz/getting-started/new-project-guide/go-lang/#buildsh
-compile_go_fuzzer ./libcontainer/system FuzzUIDMap id_map_fuzzer linux
-compile_go_fuzzer ./libcontainer/user FuzzUser user_fuzzer
-compile_go_fuzzer ./libcontainer/configs FuzzUnmarshalJSON configs_fuzzer
+compile_go_fuzzer github.com/opencontainers/runc/libcontainer/system FuzzUIDMap id_map_fuzzer linux,gofuzz
+compile_go_fuzzer github.com/opencontainers/runc/libcontainer/user FuzzUser user_fuzzer
+compile_go_fuzzer github.com/opencontainers/runc/libcontainer/configs FuzzUnmarshalJSON configs_fuzzer
```

Comment:
Wonder if we need `gofuzz` tag for these lines as well

Reply:
It works fine as is locally for me.

Reply:
gofuzz tag is automatically appended by compile_go_fuzzer when we don't specify any build tags